src/sentry/integrations/tasks/sync_status_inbound.py

```diff
@@ -204,15 +207,16 @@ def sync_status_inbound(
 ) -> None:
     from sentry.integrations.mixins import ResolveSyncAction
 
-    integration = integration_service.get_integration(
-        integration_id=integration_id, status=ObjectStatus.ACTIVE
-    )
+    integration = integration_service.get_integration(integration_id=integration_id)
     if integration is None:
         raise Integration.DoesNotExist
+    elif integration.status != ObjectStatus.ACTIVE:
+        return
+
+    organization = Organization.objects.get(id=organization_id)
 
-    organizations = Organization.objects.filter(id=organization_id)
     affected_groups = list(
-        Group.objects.get_groups_by_external_issue(integration, organizations, issue_key)
+        Group.objects.get_groups_by_external_issue(integration, [organization], issue_key)
```

Comment:
nit: Using kwargs here will help make it clear why we're passing it as a list without having to dig into the implementation. Personally I just use kwargs for everything always but that might be extreme

Reply:
uh kwargs as in named parameters(idk if that's the actual name for it either)? i.e
```suggestion
        Group.objects.get_groups_by_external_issue(integration=integration, organizations=[organization], external_issue_key=issue_key)
```

Reply:
yeah, mb on jargon -- https://docs.python.org/3/glossary.html#term-argument it means `keyword argument` and that's exactly it.